src/apps/vpn/appconstants.cpp

```diff
@@ -41,7 +41,6 @@ QString AppConstants::apiUrl(ApiEndpoint endpoint) {
       {ApiEndpoint::Device, "/api/v1/vpn/device"},
       {ApiEndpoint::DeviceWithPublicKeyArgument, "/api/v1/vpn/device/%1"},
       {ApiEndpoint::DNSDetectPortal, "/api/v1/vpn/dns/detectportal"},
-      {ApiEndpoint::Feedback, "/api/v1/vpn/feedback"},
```

Comment:
File an issue to remove this code in Guardian. Not immediately, but in a couple of releases